src/hb-ot-layout.cc

```diff
@@ -69,7 +69,7 @@
  * Tests whether a face includes any kerning data in the 'kern' table.
  * Does NOT test for kerning lookups in the GPOS table.
  *
- * Return value: true if data found, false otherwise
+ * Return value: %true if data found, false otherwise
```

Comment:
Why not also `%false`?

Reply:
That's a good point (only real reason is that prior inline comments elsewhere used %true). On the other hand, there are a lot of those, sprinkled around a lot of files, so handling them all in a separate PR after the big merges are done might be nicer -- get more docs out faster, so to speak. I'm not sure how useful all the decorations are in reality anyway.

Reply:
Sounds good.